### PR TITLE
Proposal: Clean up issue labels

### PR DESCRIPTION
**Pull request by timstclair**
cAdvisor issues are currently a bit of a mess. Problems include:
- Lots of issues which might be resolved, but are still open
- Old issues with no activity in over a year
- Most issues are unassigned, so it's unclear whether anyone is actually working on / owning the issue
- Most issues are unlabeled, and the current labels are not particularly descriptive

This proposal addresses the last point by adding a document describing the new labels. In addition to the new labels described, I'm proposing the following changes:
- Remove: `invalid`, `wontfix` (merged into `closed/infeasible` - less harsh sounding). (No issues are currently labeled with either of those)
- Prefix: `bug` and `enhancement` with `kind/`
- Rename `question` to `kind/support`

The list of labels is definitely not complete (particularly the `area/` labels). LMK if there are other labels you would like to see added.

If this PR is accepted, I will go ahead and make the proposed label changes (and start labeling issues).

Additionally, I created a `docs/development` subdirectory, and moved cAdvisor development related docs into that directory.

### CODE REVIEW

File: docs/development/issues.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# GitHub Issue tracking cAdvisor
```
Comment (vishh) [inline docs/development/issues.md]: We should include a note on assigning issues to contributors outside of the Maintainers list.
A simple comment might suffice.

**Comment by vishh** [comment]
LGTM. Thanks @timstclair !

File: docs/development/issues.md (hunk 1 above)
Comment (timstclair) [inline docs/development/issues.md]: Done: added `community-assigned` label.

**Comment by vishh** [comment]
@pwittrock: gcloud is still being flaky :( Time to switch to pure ssh I guess.

**Comment by vishh** [comment]
LGTM

**Comment by vishh** [comment]
I will let @jimmidyson and @pwittrock review this PR before merging.

**Comment by jimmidyson** [comment]
LGTM

**Comment by pwittrock** [comment]
Looks good.

**Comment by timstclair** [comment]
@k8s-bot test this

**Comment by timstclair** [comment]
Anyone know how to refresh the jenkins auth credentials?

**Comment by timstclair** [comment]
And any objections to ignore the tests for this? (It's just docs)

**Comment by pwittrock** [comment]
More credentials failures :(

**Comment by vishh** [comment]
Let's ignore the tests

**Comment by timstclair** [comment]
Thanks. I added the new labels, and made a quick pass (a lot more is needed) to add them to some issues. In doing so, I got a few ideas for other labels that might be useful. I'll follow up to add them later, but I'm listing them now so I remember :)
- `area/runtime` (e.g. docker, rkt, hyper, etc.)
- `metrics/{cpu,memory,network,fs,etc.}` for issues related to specific metrics

**Pull request merged**